version: '3'
services:
  sync:
    image: waisbrot/wait
    depends_on:
      - auth
      - postgres
    environment:
      TARGETS: "auth:9497 postgres:5432"
      TIMEOUT: "${INTEGRATION_TEST_TIMEOUT}"
  auth:
    image: wazopbx/wazo-auth-tests
    ports:
      - "9497"
    environment:
      XIVO_UUID: the-predefined-xivo-uuid
    volumes:
      - "../../..:/usr/src/wazo-auth"
      - "./etc/wazo-auth/config.yml:/etc/wazo-auth/config.yml"
      # - "${LOCAL_GIT_REPOS}/xivo-lib-python/xivo:/usr/local/lib/python2.7/site-packages/xivo"
      # - "${LOCAL_GIT_REPOS}/xivo-bus/xivo_bus:/usr/local/lib/python2.7/site-packages/xivo_bus"
    command: "wazo-auth"
  postgres:
    image: wazopbx/wazo-auth-db
    expose:
      - "5432"
